java/compaction/compaction-core/src/main/java/sleeper/compaction/job/CompactionJob.java

```diff
@@ -179,54 +135,35 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-        CompactionJob compactionJob = (CompactionJob) o;
-        return isSplittingJob == compactionJob.isSplittingJob &&
-                Objects.equals(splitPoint, compactionJob.splitPoint) &&
-                Objects.equals(dimension, compactionJob.dimension) &&
-                Objects.equals(tableName, compactionJob.tableName) &&
-                Objects.equals(jobId, compactionJob.jobId) &&
-                Objects.equals(inputFiles, compactionJob.inputFiles) &&
-                Objects.equals(outputFile, compactionJob.outputFile) &&
-                Objects.equals(outputFiles, compactionJob.outputFiles) &&
-                Objects.equals(partitionId, compactionJob.partitionId) &&
-                Objects.equals(iteratorClassName, compactionJob.iteratorClassName) &&
-                Objects.equals(iteratorConfig, compactionJob.iteratorConfig);
+        CompactionJob that = (CompactionJob) o;
+        return isSplittingJob == that.isSplittingJob && dimension == that.dimension && Objects.equals(tableId, that.tableId) && Objects.equals(jobId, that.jobId) && Objects.equals(inputFiles, that.inputFiles) && Objects.equals(outputFile, that.outputFile) && Objects.equals(outputFiles, that.outputFiles) && Objects.equals(childPartitions, that.childPartitions) && Objects.equals(partitionId, that.partitionId) && Objects.equals(splitPoint, that.splitPoint) && Objects.equals(iteratorClassName, that.iteratorClassName) && Objects.equals(iteratorConfig, that.iteratorConfig);
```

Comment:
This line is too long - can it be reformatted so that it's similar to the way it was before?

Reply:
Sure. It was automatically generated, maybe we should change our settings to make it generate a more readable version.